### PR TITLE
Feat: footer

### LINKED ISSUE #6: Build out footer
## Content
* [x] Check-in with Haley
* [x] @AnthonyRollins Respond to questions and add updates to [lo-fi wireframes](https://docs.google.com/document/d/1iIWa7jx5wzGU2RjyufJ_X13Q9RE5eAVYNpQ4aDW8E1E/edit#)
* [x] Haley making edits to [content](https://docs.google.com/document/d/1iIWa7jx5wzGU2RjyufJ_X13Q9RE5eAVYNpQ4aDW8E1E/edit#) strategy
* [ ] ~Finalize homepage [content](https://docs.google.com/document/d/1iIWa7jx5wzGU2RjyufJ_X13Q9RE5eAVYNpQ4aDW8E1E/edit#) strategy~ Not needed for this ticket
* [x] Send footer content to designer
## Design
* [ ] ~Vyki to check in with Kristi~
* [x] Send designer final content / strategy
* [x] Approve final design drafts for footer
## Implementation
- [ ] Develop footer based on design specs

## Resources 
- Prior art / initial [Figma design](https://www.figma.com/file/fkRyBpWGDyaBh1kzaXIljo/Compiler-website?node-id=0%3A1) (not final, but to give an idea)
- [Lo-Fi Wireframe ](https://docs.google.com/document/d/1iIWa7jx5wzGU2RjyufJ_X13Q9RE5eAVYNpQ4aDW8E1E/edit#)

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #6 

This PR adds a footer to the Compiler website according to the design in Figma ([desktop](https://www.figma.com/file/2gq3WJFdRFDefyHcgFMUGt/Current-compiler.la?node-id=8%3A49&t=MzUx1EjcCJdxL5In-4) and [mobile](https://www.figma.com/file/2gq3WJFdRFDefyHcgFMUGt/Current-compiler.la?node-id=309%3A1615&t=MzUx1EjcCJdxL5In-4)).

### CODE REVIEW

**Review by AnthonyRollins — CHANGES_REQUESTED** [review]
Here's a copy of my review from #31: 

@angela-tran Answers to your proposals are below, but I was curious if there is a mobile screenshot I can get of the new footer or if I can wait until the preview link is generated. Also, I remembered our Figma files don't have mobile comps associated with them, so curious about what the footer looks like on mobile.

Contact section:
I like the idea of a content section, but going back and forth, if "Contact Us" should mirror the same color as the other headers, "NAICS Codes," "Certifications," etc.

 Change font color for Contact Us to #01B4CB
[hello@compiler.la](mailto:hello@compiler.la) link:
Let's keep it white, and I believe we had it on-hover change to the compiler green #88B440. The font change makes sense to me, so everything is the same.

Section font:
No special styling works for me. (16px font).

Social media icons:
I believe we changed the spacing to match the entire width of the "Contact Us" column, but I don't think we need it to. Curious if this helps on mobile or if there is any strong reason to match the spacing of the column @angela-tran @thekaveman

**Comment by angela-tran** [comment]
@AnthonyRollins 

**Contact section**: I don't feel too strongly either way, so I can definitely make "Contact Us" match the color of the other headings. I'll make the font match too.

**Social media icons**: I also don't think we need it to match the column width, but that's just me 😄

**Comment by angela-tran** [comment]
Also @AnthonyRollins , you should be able to use the preview to see how the site behaves on mobile or when the browser window is resized on desktop: https://deploy-preview-32--compilerla.netlify.app/

**Comment by AnthonyRollins** [comment]
> **Social media icons**: I also don't think we need it to match the column width, but that's just me 😄

@angela-tran I don't think we need to either, so I would vote that we keep it like how it was before.

**Comment by thekaveman** [comment]
I think I'd like us to apply Bootstrap or some grid system here and fix the collapse down to mobile - stacking down to a single column seems like a better UX than squeezing everything together across multiple columns.

I also think we should reorder the certs back to how they originally were: Certifications, Contract Vehicles, NAISC

@AnthonyRollins and I also talked about maybe removing the Contact Us header/column altogether and moving that information more linearly/inline down below the columns, above the social icons/brand mark.

Will play around in Figma for all this... maybe this weekend? 🤷

**Comment by angela-tran** [comment]
Marking this as draft while work is being done on #33

**Comment by thekaveman** [comment]
Removed myself as a reviewer for now to reduce noise. Please re-request when this is ready to go!

**Comment by angela-tran** [comment]
Hi @AnthonyRollins and @thekaveman!

I've made some changes to make the footer better match the designs in Figma, including the mobile design that @segacy1 created.

There are some comments earlier in this PR where we discussed different ideas about fonts, colors, etc., but since they are from quite a while ago, I decided to just reference the current Figma designs to keep it simple.

One thing I did incorporate from @thekaveman's [comment](https://github.com/compilerla/compiler.la/pull/32#issuecomment-1264121401) though was:
> I also think we should reorder the certs back to how they originally were: Certifications, Contract Vehicles, NAISC

**Comment by angela-tran** [comment]
One more note: I made the minimal amount of changes needed here to get the layout working; there is a new, separate ticket for getting us on Bootstrap (#36).

File: styles/base.css [hunk 1]
```diff
@@ -24,7 +24,7 @@ body {
     display: flex;
     flex-direction: column;
 
-    padding: 30px;
+    padding: 16px;
```
Comment (thekaveman) [inline styles/base.css:27]: I'm curious about this change? Personally I like a little more breathing room around the edges.

**Review by thekaveman — COMMENTED** [review]
Generally looks a lot better on mobile! One small question.

**Comment by AnthonyRollins** [comment]
@angela-tran Looks good to me

**Comment by segacy1** [comment]
hey @angela-tran, this looks great! Just one idea—could we add a new line break between the footer and the rest of the page? It’s 1px, full-width, in the new gray-4. I’ll tag you in figma so you can see what I mean, but attached a picture.
Screen Shot 2022-12-19 at 2 38 27 PM

File: styles/base.css (hunk 1 above)
Comment (angela-tran) [reply styles/base.css]: This change was to match the spacing specified in Figma for mobile. I agree though that 16px looks a bit too close to the edge for desktop.

In [f0d43fa](https://github.com/compilerla/compiler.la/pull/32/commits/f0d43fa88ab4a589eb9b837da2da2025bf660e97), I've put back the 30px padding for desktop and added to the existing media-query so mobile has 16px padding.

Comment (segacy1) [reply styles/base.css]: I'll update in Figma!

**Comment by angela-tran** [comment]
@AnthonyRollins Yep, you are looking at the correct link. I didn't notice that the `Hover` and `Pressed` colors in Figma are a little different from the current site so thanks for pointing that out. I am not sure though if @segacy1 is ready for those colors to be implemented.

I've made a new separate issue (https://github.com/compilerla/compiler.la/issues/37) for implementing new style/brand changes so that this PR can focus on getting the footer in.

**Comment by angela-tran** [comment]
> hey @angela-tran, this looks great! Just one idea—could we add a new line break between the footer and the rest of the page? It’s 1px, full-width, in the new gray-4.

👍 Done

**Comment by segacy1** [comment]
Thanks @angela-tran and @AnthonyRollins! Yes, I'm playing around with the colors a bit for the job page design, so they'll likely be a work-in-progress until our next check-in.

**Review by AnthonyRollins — APPROVED** [review]
Looks good to me

**Pull request merged**